src/lib/validate-raw-data.js

```diff
@@ -53,6 +53,7 @@ async function validate_raw_data_all() {
 
         // Step 2 - process each item of data we got
         for (var raw_data of rows) {
+            console.log("validating:", raw_data.id)
```

Comment:
I assume this has been intentionally left here for logging the state before potential failure?

Reply:
yeah, am leaving it in for now at least. We don't have a consistent logging policy/method anyway.